kinto/core/initialization.py

```diff
@@ -132,7 +133,12 @@ def on_new_response(event):
         # Add backoff in response headers.
         backoff = config.registry.settings['backoff']
         if backoff is not None:
-            event.response.headers['Backoff'] = str(backoff)
+            backoff_percentage = config.registry.settings['backoff_percentage']
```

Comment:
Do we need to convert the value to integer?

Reply:
I don't believe so, otherwise
```python
if random.random() > (backoff_percentage / 100.0)
```
would have raised a `TypeError` exception when the tests ran

Reply:
Well in your test you test with a mock of a integer, the question is did you try it with an actually config file?

Reply:
I agree with @Natim, an explicit cast to int or float would not harm here